tools/scripts/acr-build.ps1

```diff
@@ -79,7 +79,7 @@ else {
         # dev feature builds
         $namespace = $namespace.Replace("feature/", "")
     }
-    elseif ($namespace.StartsWith("release/") -or ($namespace -eq "master")) {
+    elseif ($namespace.StartsWith("release") -or ($namespace -eq "master")) {
```

Comment:
I would recommend testing for release/ and releases separately.  I would also remove "master", which then could be replaced by "releases".   That way it is clear to the reader which namespace is meant here.

Reply:
Agreed on removing "master". I can remove from here.
Since we were using StartsWith, shouldn't "release" suffice? What am I missing with 
```suggestion
    elseif ($namespace.StartsWith("release") -or ($namespace -eq "releases")) {
```
vs 
```suggestion
    elseif ($namespace.StartsWith("release")) {
```